.github/workflows/test.yml

```diff
@@ -46,19 +50,7 @@ jobs:
         run: bin/rails db:schema:load
       # Add or replace test runners here
       - name: Run tests
-        run: bin/rake
-
-  lint:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v3
-      - name: Install Ruby and gems
-        uses: ruby/setup-ruby@d2b39ad0b52eca07d23f3aa14fdf2a3fcc1f411c # v1.117.0
-        with:
-          bundler-cache: true
-      - name: Install frontend dependencies
-        run: npm ci
```

Comment:
Can we also delete https://github.com/alphagov/forms-admin/blob/main/package.json#L18  if we arent calling `npm ci`?

Reply:
No because we're still calling `npm ci` on line 43 above, its just that we're no longer doing it twice.